[2022-04-07 19:25:13,064] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task scheduled__2017-06-01T00:00:00+00:00 [queued]>
[2022-04-07 19:25:13,194] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task scheduled__2017-06-01T00:00:00+00:00 [queued]>
[2022-04-07 19:25:13,194] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:25:13,194] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 19:25:13,194] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:25:14,266] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task> on 2017-06-01 00:00:00+00:00
[2022-04-07 19:25:14,384] {standard_task_runner.py:52} INFO - Started process 4128 to run task
[2022-04-07 19:25:14,426] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task', 'scheduled__2017-06-01T00:00:00+00:00', '--job-id', '12924', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmps2hrwq5j', '--error-file', '/tmp/tmpqxr2_k6d']
[2022-04-07 19:25:14,460] {standard_task_runner.py:80} INFO - Job 12924: Subtask create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task
[2022-04-07 19:25:14,695] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task scheduled__2017-06-01T00:00:00+00:00 [running]> on host 2c54f2187010
[2022-04-07 19:25:15,064] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2017-06-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2017-06-01T00:00:00+00:00
[2022-04-07 19:25:15,070] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 19:25:15,119] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 19:25:15,150] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_badges_bq_create_badges_stack_overflow_data_partitioned_table_task_2017_06_01T00_00_00_00_00_b1c0cc38ef496cae74c06bbb38f40565
[2022-04-07 19:25:15,892] {taskinstance.py:1774} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2153, in execute
    job = self._submit_job(hook, job_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2119, in _submit_job
    retry=self.result_retry,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 439, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1560, in insert_job
    job.result(timeout=timeout, retry=retry)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1498, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1488, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 728, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/future/polling.py", line 135, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Could not cast literal "date" to type TIMESTAMP at [1:102]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_badges_bq_create_badges_stack_overflow_data_partitioned_table_task_2017_06_01T00_00_00_00_00_b1c0cc38ef496cae74c06bbb38f40565

[2022-04-07 19:25:15,946] {taskinstance.py:1288} INFO - Marking task as FAILED. dag_id=load_to_BigQuery, task_id=create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task, execution_date=20170601T000000, start_date=20220407T192513, end_date=20220407T192515
[2022-04-07 19:25:16,207] {standard_task_runner.py:98} ERROR - Failed to execute job 12924 for task create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task (400 Could not cast literal "date" to type TIMESTAMP at [1:102]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_badges_bq_create_badges_stack_overflow_data_partitioned_table_task_2017_06_01T00_00_00_00_00_b1c0cc38ef496cae74c06bbb38f40565
; 4128)
[2022-04-07 19:25:16,269] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-04-07 19:25:16,624] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-07 22:18:12,546] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task scheduled__2017-06-01T00:00:00+00:00 [queued]>
[2022-04-07 22:18:12,680] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task scheduled__2017-06-01T00:00:00+00:00 [queued]>
[2022-04-07 22:18:12,680] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:18:12,680] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 22:18:12,680] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:18:13,217] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task> on 2017-06-01 00:00:00+00:00
[2022-04-07 22:18:13,234] {standard_task_runner.py:52} INFO - Started process 6305 to run task
[2022-04-07 22:18:13,275] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task', 'scheduled__2017-06-01T00:00:00+00:00', '--job-id', '14169', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmp5trlynjh', '--error-file', '/tmp/tmphcc0qkfz']
[2022-04-07 22:18:13,276] {standard_task_runner.py:80} INFO - Job 14169: Subtask create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task
[2022-04-07 22:18:13,568] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task scheduled__2017-06-01T00:00:00+00:00 [running]> on host 36107d30f64e
[2022-04-07 22:18:13,843] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2017-06-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2017-06-01T00:00:00+00:00
[2022-04-07 22:18:13,845] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 22:18:13,895] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 22:18:13,921] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_badges_bq_create_badges_stack_overflow_data_partitioned_table_task_2017_06_01T00_00_00_00_00_6e8c38711700381230fcda94e2666920
[2022-04-07 22:18:16,637] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=load_to_BigQuery, task_id=create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task, execution_date=20170601T000000, start_date=20220407T221812, end_date=20220407T221816
[2022-04-07 22:18:16,718] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-07 22:18:16,808] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
